website_wow/static/src/js/website_wow.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2017 LasLabs Inc.
+   License GPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+*/
+
+odoo.define('website_wow.website_wow', function (require) {
+    'use strict';
+
+    var base = require('web_editor.base');
+    var core = require('web.core');
+
+    var WebsiteWow = {
```

Comment:
The code docs for the section I'm describing:

```
+    How to define an animable element in your theme
 +    ===============================================
 +    <xpath expr="//t[@t-set='o_animate_elements']" position="after">
 +        <t t-set="o_animate_elements" t-value="o_animate_elements + ['.myclass']"/>
 +    </xpath>
```

I need to join the `o_animate_elements` template variable into a selector for use. In order to go with the `web.Class` design, I would need my `selector` class attribute to be based off of this template variable. Do you know how to access this XML namespaced variable in my class?